cmd/cmd.go

```diff
@@ -182,9 +182,9 @@ func flagsToLogFields(flags *pflag.FlagSet) []z.Field {
 	return fields
 }
 
-// redact returns a redacted version of the given flag value. It currently supports redacting
+// Redact returns a redacted version of the given flag value. It currently supports redacting
 // passwords in valid URLs provided in ".*address.*" flags and redacting auth tokens.
-func redact(flag, val string) string {
+func Redact(flag, val string) string {
```

Comment:
nit: can just copy this function to testutil/promrated/promrated.go as this not a "big" function

Reply:
Agreed, copied with slight modification.